src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/permission/PermissionEvaluatorService.java

```diff
@@ -67,18 +67,18 @@ private boolean evaluateAccess(Map<String, CamundaVariable> variables,
                                    List<PermissionTypes> permissionsRequired) {
         boolean hasAccess;
 
-        log.debug("Evaluating access for {}", roleAssignment);
+        log.info("Evaluating access for {}", roleAssignment);
         // 1. Always Check Role name has required permission
         hasAccess = hasRolePermission(roleAssignment.getRoleName(), variables, permissionsRequired);
 
-        log.debug("RoleName permission check {}", hasAccess);
+        log.info("RoleName permission check {}", hasAccess);
```

Comment:
we probably don't want all this noise printing unless enabled as when performing a search this could print thousands of lines